src/nvim/rbuffer.c

```diff
@@ -15,7 +15,7 @@ RBuffer *rbuffer_new(size_t capacity)
   FUNC_ATTR_WARN_UNUSED_RESULT FUNC_ATTR_NONNULL_RET
 {
   if (!capacity) {
-    capacity = 0xffff;
+    capacity = 0x10000;
```

Comment:
Suggested buffer size by libuv and linux pipe capacity (also a multiple of the atomic pipe buffer size (4096)).